questionnaire-ko.json

```diff
@@ -1,434 +1,434 @@
+              "specReference": [
                 "3.2",
-            	"3.2.1"
-              ]
-            },
-            "3.c.iv": {
-              "correctAnswer": "Yes",
-              "question": "     Containing modified open source;",
-              "number": "3.c.iv",
-              "type": "YES_NO",
-              "specReference": [
+            	"3.2.1"
+              ]
+            },
+            "3.c.iv": {
+              "correctAnswer": "네",
+              "question": "     Copyleft 의무를 발생시킬 수 있는 다른 오픈 소스와 통합;",
+              "number": "3.c.iv",
+              "type": "YES_NO",
+              "specReference": [
                 "3.2",
-            	"3.2.1"
-              ]
-            },
-            "3.c.v": {
-              "correctAnswer": "Yes",
-              "question": "     Containing open source or other software under incompatible licenses for interaction with other components in the Supplied Software;",
-              "number": "3.c.v",
-              "type": "YES_NO",
-              "specReference": [
+            	"3.2.1"
+              ]
+            },
+            "3.c.v": {
+              "correctAnswer": "네",
+              "question": "     수정한 오픈소스를 포함;",
+              "number": "3.c.v",
+              "type": "YES_NO",
+              "specReference": [
                 "3.2",
-            	"3.2.1"
-              ]
-            },
-            "3.c.vi": {
-              "correctAnswer": "Yes",
-              "question": "     Containing open source with attribution requirements.",
-              "number": "3.c.vi",
-              "type": "YES_NO",
-              "specReference": [
+            	"3.2.1"
+              ]
+            },
+            "3.c.vi": {
+              "correctAnswer": "네",
+              "question": "     공급 대상 소프트웨어 내에서 상호 작용하는 다른 컴포넌트와 호환되지 않는 라이선스 하의 오픈소스 또는 기타 소프트웨어를 포함;",
+              "number": "3.c.vi",
+              "type": "YES_NO",
+              "specReference": [
                 "3.2",
-            	"3.2.1"
-              ]
-            }
-          }
-        }
-      ]
-    },
-    {
-      "name": "Part 4",
-      "title": "4. Compliance artifact creation and delivery",
-      "questions": [
-        {
-          "correctAnswer": "Yes",
-          "question": "Do you have a documented procedure describing the process for ensuring the Compliance Artifacts are distributed with Supplied Software as required by the Identified Licenses?",
-          "number": "4.a",
-          "type": "YES_NO",
-          "specReference": [
-            "4.1",
-            "4.1.1"
-          ]
-        },
-        {
-          "correctAnswer": "Yes",
-          "question": "Do you have a documented procedure for archiving copies of Compliance Artifacts for the Supplied Software?",
-          "number": "4.b",
-          "type": "YES_NO",
+            	"3.2.1"
+              ]
+            }
+          }
+        }
+      ]
+    },
+    {
+      "name": "Yes",

```

Comment:
"name" is incorrect - it should be "Part 4"

Reply:
PR later.